compose/const.py

```diff
@@ -57,6 +58,7 @@
     COMPOSEFILE_V3_5: '1.30',
     COMPOSEFILE_V3_6: '1.36',
     COMPOSEFILE_V3_7: '1.38',
+    COMPOSEFILE_V3_8: '1.40',
```

Comment:
Maybe I misunderstand this constant, but afaik 3.8 do not add any new field to require an API upgrade vs 3.7, does it?

Reply:
Yes, I set it back to 1.38. This should be the minimum API version we need for the v3.8 schema.